circle.yml

```diff
@@ -0,0 +1,42 @@
+machine:
+  services:
+    - docker
+
+dependencies:
+  override:
+    - docker info
+    - |
+      printf '{"version":{"hash":"%s","version":"%s","source":"https://github.com/%s/%s","build":"%s"}}\n' \
+        "$CIRCLE_SHA1" \
+        "$CIRCLE_TAG" \
+        "$CIRCLE_PROJECT_USERNAME" \
+        "$CIRCLE_PROJECT_REPONAME" \
+        "$CIRCLE_BUILD_URL" \
+        | tee config/version.json version.json
+    - docker build -f Dockerfile-build -t fxa-oauth-console:build .
+
+test:
+  override:
+    - /bin/true
+
+deployment:
+  hub_latest:
+    owner: mozilla
+    branch: /^(master|feature.*|dockerpush.*)$/
+    commands:
+      - "[ ! -z $DOCKERHUB_REPO ]"
+      - docker login -e "$DOCKER_EMAIL" -u "$DOCKER_USER" -p "$DOCKER_PASS"
+      - "docker tag fxa-oauth-console:build ${DOCKERHUB_REPO}:latest"
```

Comment:
this is missing custum CIRCLE branch logic, all branches wil be called  'latest'